server/graphql/v2/query/LatestChangelogPublishDateQuery.js

```diff
@@ -0,0 +1,41 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql';
+import { GraphQLDateTime } from 'graphql-iso-date';
+
+import cache, { fetchCollectiveId } from '../../../lib/cache';
+import sequelize from '../../../lib/sequelize';
+import models from '../../../models';
+
+const LatestChangelogPublishDateQuery = {
+  type: GraphQLDateTime,
+  args: {
+    collectiveSlug: {
```

Comment:
Doesn't make sense to have a parameter here (`collectiveSlug`) and then hardcode a cache key that doesn't take into account the parameter. This needs to be reviewed.

Reply:
@znarf : Ah yes, don't think there's a need to pass the parameter to grab the latest publish date. I've removed this and refactored the code. 👍🏽 🐨